code/src/components/js/HeartButton.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+
+import "../css/heartButton.css";
+
+export const HeartButton = ({ onLiked, heart, heartId }) => {
+  const handleClick = () => {
+    fetch(
+      `https://happy-thoughts-technigo.herokuapp.com/thoughts/${heartId}/like`,
+      {
+        method: "POST",
+        body: "",
```

Comment:
No need for body over here, all essential data are being passed in the URL in line 8